src/js/index.js

```diff
@@ -51,6 +51,15 @@ formAdd.addEventListener('submit', (e) => {
   input.value = '';
 });
 
+formSearch.addEventListener('input', (e) => {
+  e.preventDefault();
+  setTimeout(() => {
+    taskMenager.searchTasks(e.target.value);
+    renderTasks(taskMenager.todoTasks, todoListing);
+    renderTasks(taskMenager.doneTasks, doneListing);
+  }, 800);
```

Comment:
U must create variable to stored this timeout and on each input change clear this timeout and setting up again